src/server.h

```diff
@@ -1150,6 +1150,7 @@ typedef struct {
 typedef struct client {
     uint64_t id;            /* Client incremental unique ID. */
     uint64_t flags;         /* Client flags: CLIENT_* macros. */
+    uint64_t reset_flags;   /* flags to reset in persistent clients after each execution, generally set by RM_Call */
```

Comment:
we previously had attempts to make the important fields of the client struct fit in the same cache line.
considering this one isn't highly used, let's move it further down (after resp or db)

Reply:
done